### ISSUE TITLE
Remove divider lines in Drawing Lab

### ISSUE THREAD

**Issue by damithc** [PowerPointLabs/PowerPointLabs#861]
Some of these horizontal dividers align with tabs and makes the tab structure a bit hard to follow. We may not need horizontal dividers if we use some extra spacing to separate button groups.
![image](https://cloud.githubusercontent.com/assets/1673303/12071133/f30f67be-b0d5-11e5-982b-c984c8182ebb.png)

**Comment by vhazali** [comment]
Is this only pertaining to the Main tab? If so I believe this issue is also fixed.
